x-pack/plugins/canvas/canvas_plugin_src/renderers/embeddable/input_type_to_expression/embeddable.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { encode } from '../../../../common/lib/embeddable_dataurl';
+import { EmbeddableInput } from '../../../expression_types';
+
+export function toExpression(input: EmbeddableInput, embeddableType: string): string {
+  const expressionParts = [] as string[];
+
+  expressionParts.push('embeddable');
```

Comment:
since there's nothing conditional here, can you just build this as one big string instead of doing the parts+join.  

```js
return `embeddable config="${encode(input)}" type="${embeddableType}"`
```